projects/angular/storyteller/src/app/routes/feed/feed.component.ts

```diff
@@ -65,6 +65,12 @@ export class FeedComponent {
 
   // Filters stories by project id
   loadProjectStories(projectId: string): void {
-    // TODO: implement this
+    if (projectId !== 'all') {
+      this.easyStories = this.easyStories.filter(story => {
```

Comment:
Can we share the code?
E.g. to make method inside StoryService `getFilteredStories(projectId)`